src/components/label/InfoIconWithTooltip.js

```diff
@@ -11,7 +11,7 @@ type Props = {
 };
 
 const InfoIconWithTooltip = ({ className = '', iconProps, tooltipText }: Props) => (
-    <span key="infoIcon" className={`${className} tooltip-icon-container`}>
+    <span key="infoIcon" className={`${className} bdl-Tooltip-iconContainer`}>
```

Comment:
This class name is used outside of the Tooltip component and should have a different component-specific prefix, such as `bdl-InfoIcon`.